File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_AutoTextUsers.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_AutoTextUsers } from './Sys_AutoTextUsers.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_AutoTextUsersService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'AutoText_Id', 'User_Id'];

    List: Sys_AutoTextUsers[] | undefined;
    ListFilter: Sys_AutoTextUsers[] | undefined;
    FormData!: Sys_AutoTextUsers;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_AutoTextUsers";
    }
}

File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_AutoTextUsersService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_AutoTextUsersService : BaseService<Sys_AutoTextUsers, ISys_AutoTextUsersRepository>
    , ISys_AutoTextUsersService
    {
    private readonly ISys_AutoTextUsersRepository _Sys_AutoTextUsersRepository;
    public Sys_AutoTextUsersService(ISys_AutoTextUsersRepository Sys_AutoTextUsersRepository) : base(Sys_AutoTextUsersRepository)
    {
    _Sys_AutoTextUsersRepository = Sys_AutoTextUsersRepository;
    }
    }
    }

